src/nvim/eval.c

```diff
@@ -512,6 +512,8 @@ void eval_init(void)
       assert(false);
     }
   }
+  vimvars[VV_VERSION].vv_nr = VIM_VERSION_100;
```

Comment:
Is there a reason not to put this at line 497, which is where it is in the original patch?